flake.nix

```diff
@@ -15,6 +15,25 @@
         pkgs = import "${nixpkgs}" {
           inherit system;
           config.allowUnfree = true;
+          overlays = [
+            (final: prev: {
+              nix = prev.nix.overrideAttrs
+                (finalAttrs: prevAttrs: {
+                  patches = (prevAttrs.patches or [ ]) ++ [
+                    (final.fetchpatch {
+                      name = "include-untracked-files.patch";
+                      url = "https://github.com/NixOS/nix/pull/9352.patch";
+                      excludes = [ "tests/functional/*" ];
+                      hash = "sha256-JPoQSXFKex6wFx2AoOuMTY0PvzdZ9R5wCqcsT+13i7k=";
+                    })
+                  ];
+                  # Set big-parallel, this allows us to build nix remotely
+                  # on machines where only big-parallel derivations are built
+                  # remotely.
```

Comment:
I'm tempted to delete this comment. If a reader has no idea what `big-parallel` is, this comment is not that helpful without actually googling and reading up on `big-parallel`. But for that, we don't need a comment. WDYT?